cerberus/kubernetes/client.py

```diff
@@ -30,29 +30,46 @@ def initialize_clients(kubeconfig_path, chunk_size, timeout):
     kubeconfig_path_global = kubeconfig_path
 
 
+def list_continue_helper(func, **keyword_args):
+    ret_overall = []
+    try:
+        pretty_arg = keyword_args.get("pretty", False)
+        ret = func(pretty=pretty_arg, limit=request_chunk_size)
```

Comment:
Shouldn't we pass all kwargs to the called function? That would make it flexible to call any Kubernetes list function with parameters.

Reply:
do you mean doing something like the below and set any defaults before the list_continue_helper is called? 

ret = func(**keyword_args)